lib/ret/discord_client.ex

```diff
@@ -43,11 +43,17 @@ defmodule Ret.DiscordClient do
     |> Poison.decode!()
   end
 
-  def member_of_channel?(account_id, %Ret.HubBinding{} = hub_binding) do
+  def has_permission?(nil, _, _), do: false
+
+  def has_permission?(%Ret.OAuthProvider{} = oauth_provider, %Ret.HubBinding{} = hub_binding, permission) do
+    oauth_provider.provider_account_id |> has_permission?(hub_binding, permission)
+  end
+
+  def has_permission?(account_id, %Ret.HubBinding{} = hub_binding, permission) when is_binary(account_id) do
```

Comment:
i'd rename `account_id` to `provider_account_id`, since `account_id` implied to me it was expecting a `Ret.Account` id